internal/services/oracle/oracle_cloud_vm_cluster_data_source.go

```diff
@@ -466,6 +472,7 @@ func (d CloudVmClusterDataSource) Read() sdk.ResourceFunc {
 					state.CloudExadataInfrastructureId = props.CloudExadataInfrastructureId
 					state.ClusterName = pointer.From(props.ClusterName)
 					state.CompartmentId = pointer.From(props.CompartmentId)
+					state.ComputeModel = string(pointer.From(props.ComputeModel))
```

Comment:
we have a helper for this
```suggestion
					state.ComputeModel = pointer.FromEnum(props.ComputeModel)
```